build-rirods.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# build image
+sudo docker image build --tag rirods:build .
+
+# local source and library paths
+SRCPATH=$(R -e "getwd()"| grep /)
+LIBPATH=$(R -e ".libPaths()[1]" | grep /)
```

Comment:
this requires R is installed on the machine running docker... I was hoping to have little/no dependencies on the host machine.

Reply:
My main idea was to help the development of the R package. So, for that reason, I use volumes to execute R code (devtools) in the container but have my files updated. This file is also included in the buildignore.